### PR TITLE
[zh-cn]: create the translation of MathML global property `displaystyle`

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20746

### CODE REVIEW

File: files/zh-cn/web/mathml/global_attributes/displaystyle/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: displaystyle
+slug: Web/MathML/Global_attributes/displaystyle
+l10n:
+  sourceCommit: 8eece0b998c23e8ea35f936d7371a169974130f5
+---
+
+{{MathMLRef}}
+
+**`displaystyle`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes) 是一个布尔值，用于设置 MathML 元素的 [math-style](/zh-CN/docs/Web/CSS/math-style)。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/displaystyle/index.md:10]:
```suggestion
**`displaystyle`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes)是一个布尔值，用于设置 MathML 元素的 [math-style](/zh-CN/docs/Web/CSS/math-style)。
```

File: files/zh-cn/web/mathml/global_attributes/displaystyle/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## 示例
+
+在这个示例中，使用 [munder](/zh-CN/docs/Web/MathML/Element/munder) 元素将脚本 "A" 附加到基础的 "∑" 上。默认情况下，求和符号是以从其父元素继承的 [font-size](/zh-CN/docs/Web/CSS/font-size) 渲染的，并且 A 作为缩小的下标。通过显式设置 `displaystyle="true"` 属性，求和符号将会变大，并且 "A" 变成一个下标。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/displaystyle/index.md:14]:
```suggestion
在这个示例中，使用 [munder](/zh-CN/docs/Web/MathML/Element/munder) 元素将脚本“A”附加到基础的“∑”上。默认情况下，求和符号是以从其父元素继承的[字体大小](/zh-CN/docs/Web/CSS/font-size)渲染的，并且 A 作为缩小的下标。通过显式设置 `displaystyle="true"` 属性，求和符号将会变大，并且“A”变成一个下标。
```

File: files/zh-cn/web/mathml/global_attributes/displaystyle/index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+## 语法
+
+```html-nolint
+<math displaystyle="true"></math>
+<math displaystyle="false"></math>
+```
+
+### 可选值
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/displaystyle/index.md:36]:
```suggestion
### 值
```

File: files/zh-cn/web/mathml/global_attributes/displaystyle/index.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- 所有 [全局属性](/zh-CN/docs/Web/MathML/Global_attributes)
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/displaystyle/index.md:53]:
```suggestion
- 所有[全局属性](/zh-CN/docs/Web/MathML/Global_attributes)
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Some suggtions for your reference.

File: files/zh-cn/web/mathml/global_attributes/displaystyle/index.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+title: displaystyle
+slug: Web/MathML/Global_attributes/displaystyle
+l10n:
+  sourceCommit: 8eece0b998c23e8ea35f936d7371a169974130f5
+---
+
+{{MathMLRef}}
+
+**`displaystyle`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes)是一个布尔值，用于设置 MathML 元素的 [math-style](/zh-CN/docs/Web/CSS/math-style)。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/displaystyle/index.md:10]: 请等待 #20679 的确定

```suggestion
**`displaystyle`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes)是一个用于设置 MathML 元素的 [math-style](/zh-CN/docs/Web/CSS/math-style) 的布尔值。
```

**Comment by tianyeeT** [comment]
@yin1999 @JasonLamv-t @jasonren0403 :
Hi, is there anyone available to review this PR, and then we close the issue #20746.

**Comment by JasonLamv-t** [comment]
Sorry for the late. I will review this afternoon. Thxu so mcuh.

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thank you for you patience.

**Pull request merged**